static/src/stylesheets/layout/new-header/_feedback-form.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Temporary form for feedback on the desktop navigation
+ */
+
+.feedback-form {
```

Comment:
Actually this seems to be fine because its already in the right place in the markup. I checked this on safari 6. I will merge this today, but please let me know if I have missed something and I will go back and fix it